fl_aggregator/main.py

```diff
@@ -273,6 +288,17 @@ def check_proj_requests(client: Client, proj_folder: Path):
         )
 
 
+def check_has_data(client: Client, proj_folder: Path):
+    """
+    Check if the clients have data
+    """
+    fl_clients = get_all_directories(proj_folder / "fl_clients")
+    for fl_client in fl_clients:
+        print(f"Checking if the client {fl_client.name} have data")
+    
+    # TODO: check data exists (signaled by the clients)
```

Comment:
This needs to be signaled by the clients since we can't check for their `private` folder